troubleshooting.rst

```diff
@@ -51,22 +51,16 @@ No audio
 
 Updating the launcher
 ~~~~~~~~~~~~~~~~~~~~~
-Depending on your system the process may vary:
+to update the launcher on newer versions of the launcher Press the gear icon and then check for updates
 
-- Ubuntu prebuilt packages - :code:`sudo apt-get update && sudo apt-get upgrade`
+if your launcher is too old or you cant find these buttons 
+in Ubuntu prebuilt packages do sudo apt-get update && sudo apt-get upgrade 
 
-- Mac OS - Redownload the package
+and on Mac OS - Redownload the package
 
-macOS X Catalina - :code:`Could not find the game launcher..`
```

Comment:
I replaced the `Could not find the game launcher..` message,
but still applies if someone find an old version and get this error